event-management-frontend/src/components/eventCreateOrEdit/locationSection/SearchBarSuggestions.tsx

```diff
@@ -1,34 +1,32 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import useStylesSearchBar from '../../../styles/SearchBarStyle';
 import { LocationType } from '../../../model/LocationType';
 
 interface Props {
   suggestions: LocationType[];
-  suggestionSelected: (name: string) => void;
+  suggestionSelected: (loc: LocationType) => void;
   setSuggestions: (suggestion: LocationType[]) => void;
 }
 const RenderSuggestions = (props: Props) => {
   const classesSearch = useStylesSearchBar();
 
-  if (props.suggestions.length === 0) {
-    return null;
-  } else {
-    if (props.suggestions.length > 4) {
-      const firstSuggestions = props.suggestions.slice(0, 4);
+  useEffect(() => {
+    if (props.suggestions.length > 5) {
```

Comment:
This 5 should also be extracted in that constants file (`MAX_SUGGESTIONS_SHOWED` or something like that) so you can easily change it without searching for it within the code

Reply:
Fixed